CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add `roundTripTime`, `audioUplinkJitter`, and `audioDownlinkJitter` to observable metrics
```

Comment:
Does this require any browser specific documentation or existing doc updates?

Reply:
We don't have any doc for `metricsDidReceive` event for WebRTC metrics. I think it's fine.